### ISSUE TITLE
ILSVRC2012 database 

### ISSUE THREAD

**Issue by prachikashikar** [CEA-LIST/N2D2#67]
Hi,

Could you please guide me to download ILSVRC2012 database to train and test ResNet model provided with N2D2?

**Comment by Etienne-DUPUIS** [comment]
Hi, 

I had the same issue, unfortunately, Imagenet dataset is no longer publicly available on the imagenet website, one workaround if you are using the dataset for non-profit research purpose is to to download the dataset using academic torrent.
[Imagenet Download](https://academictorrents.com/collection/imagenet )
You need the "ILSVRC 2012 Training Set" and the "ILSVRC 2012 Validation Set (Object Detection)"

Then you need to extract the tar and the ones inside, [you can do it easily with a python script](https://gist.github.com/geohot/85da463f9252a64f957b).

for N2D2 the dataset should be in the format (you can set $N2D2_DATA to anything you want)
$N2D2_DATA/ILSVRC2012/train/n02747177/n02747177_10088.JPEG [...]
$N2D2_DATA/ILSVRC2012/val/ILSVRC2012_val_00000001.JPEG [...]

You also need both the files [synsets.txt](https://raw.githubusercontent.com/HoldenCaulfieldRye/caffe/master/data/ilsvrc12/synsets.txt) and [val.txt](https://git.hipert.unimore.it/EmbeddedCNN/PipeCNN/raw/dc00a0505d0f50456011cda929a10dc3d6a3d50d/data/imagenet/val.txt) at the address:
$N2D2_DATA/ILSVRC2012/synsets.txt
$N2D2_DATA/ILSVRC2012/val.txt

Best,

**Comment by prachikashikar** [comment]
Hi @Etienne-DUPUIS 

Thank you for your kind reply.  This would surely help me. :)

**Comment by olivierbichler-cea** [comment]
Closing.
